### PR TITLE
Feat/add odom publisher

### PR DESCRIPTION
**Pull request by jadhm**

### CODE REVIEW

**Comment by jadhm** [comment]
It's handy to have published Odometry messages as well as Poses. 
@tonybaltovski  Would you like to take a look at this pull request ?

**Comment by jadhm** [comment]
ping @tonybaltovski @yamokosk @abencz

File: src/rigid_body_publisher.cpp [hunk 1]
```diff
@@ -129,6 +176,18 @@ void RigidBodyPublisher::publish(ros::Time const& time, RigidBody const& body)
                    pose.pose.orientation.z,
                    pose.pose.orientation.w);
 
+  if (config.publishOdom)
+    {
```
Comment (tonybaltovski) [inline src/rigid_body_publisher.cpp:180]: Fix spacing please.

File: src/rigid_body_publisher.cpp [hunk 2]
```diff
@@ -76,7 +77,47 @@ namespace utilities
...
+      OdometryMsg.pose.pose.position.x = body.pose.position.x;
+      OdometryMsg.pose.pose.position.y = body.pose.position.z;
+      OdometryMsg.pose.pose.position.z = body.pose.position.y;
+
+      OdometryMsg.pose.pose.orientation.x = body.pose.orientation.x;
+      OdometryMsg.pose.pose.orientation.y = body.pose.orientation.z;
+      OdometryMsg.pose.pose.orientation.z = body.pose.orientation.y;
+      OdometryMsg.pose.pose.orientation.w = body.pose.orientation.w;
+    }
+    else if (coordinatesVersion < Version("2.0") && coordinatesVersion >= Version("1.7")) {
```
Comment (tonybaltovski) [inline src/rigid_body_publisher.cpp:96]: Brace should be on a new line.

**Review by tonybaltovski — CHANGES_REQUESTED** [review]
Just some minor nits.

**Review by tonybaltovski — APPROVED** [review]

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**